UIforETW/UIforETWDlg.cpp

```diff
@@ -1157,9 +1966,7 @@ void CUIforETWDlg::SaveNotesIfNeeded()
 	if (editedNotes != traceNotes_)
 	{
 		if (!traceNoteFilename_.empty())
-		{
 			WriteTextAsFile(traceNoteFilename_, editedNotes);

```

Comment:
> parentheses

Braces?

The funny thing here, is that I'm usually very much so in favor of using braces in **every single conditional**, but here I decided to follow [Google style](https://google-styleguide.googlecode.com/svn/trunk/cppguide.html#Conditionals) fair enough.